README.md

```diff
@@ -189,13 +189,13 @@ lo        Link encap:Local Loopback
 
 ## What might CNI do in the future?
 
-CNI currently covers a wide range of needs for network configuration due to it simple model and API.
+CNI currently covers a wide range of needs for network configuration due to it's simple model and API.
```

Comment:
This should be "its" (eg not possessive).

Reply:
@dcbw Ironic for a typo PR :) Updated.